picker/rofimoji.py

```diff
@@ -102,6 +104,14 @@ def parse_arguments() -> argparse.Namespace:
         metavar='FILE',
         help='Read characters from this file instead, one entry per line'
     )
+    parser.add_argument(
+        '--prompt',
+        '-r',
+        dest='prompt',
+        action='store',
+        default='😀',
```

Comment:
Could you please add one space after the emoji, like `😀 `?

Reply:
Done!